src/app/pages/home/due-soon-tasks/due-soon-tasks.component.ts

```diff
@@ -18,21 +20,35 @@ export enum ProximityDate {
 	templateUrl: './due-soon-tasks.component.html',
 	styleUrls: ['./due-soon-tasks.component.scss']
 })
-export class DueSoonTasksComponent implements OnInit {
-
-	dueSoonTasks$: Observable<Task[]>;
+export class DueSoonTasksComponent implements OnInit, OnDestroy {
+	
+	homeUpdaterSubscription: Subscription;
+	dueSoonTasks: Task[] = [];
 	proximityDate = ProximityDate;
 
 	@Input('dueDays') dueDays: number = 3;
 
 	constructor(
 		private taskService: TaskService,
 		private router: Router,
+		private changeDetectorRef: ChangeDetectorRef,
+		private homeUpdater: HomeUpdaterService,
 		private dateProximity: DateProximityPipe,
 		private sharedService: SharedService) {}
 
 	ngOnInit(): void {
-		this.dueSoonTasks$ = this.taskService.findBetweenDates(this.dueDays, this.sharedService.getUserLogged().id);
+		this.taskService.findBetweenDates(this.dueDays, this.sharedService.getUserLogged().id)
+			.subscribe(tasks => this.dueSoonTasks = tasks);
+
+		this.homeUpdaterSubscription = this.homeUpdater.getSubscriberHomeUpdate().subscribe(taskPublished => {
+			let tasksCopy = [...this.dueSoonTasks];
```

Comment:
É mais uma pergunta, do que uma sugestão ...

Por que não utilizar logo `this.dueSoonTasks.forEach()`  ao invés de criar uma variável só pra depois percorrer o vetor ? Só questão de organização mesmo ?